src/lib/Liblog/pbs_messages.c

```diff
@@ -393,6 +393,14 @@ char *msg_corelimit = "invalid value for PBS_CORE_LIMIT in pbs.conf, continuing
 char *msg_resc_busy = "Resource busy";
 
 char *msg_job_moved = "Job moved to remote server";
+char *msg_init_recovsched = "Recovered scheduler %s";
+char *msg_sched_exist = "Scheduler already exists";
+char *msg_sched_name_big = "Scheduler name is too big";
+char *msg_unknown_sched = "Unknown Scheduler";
+char *msg_no_del_sched = "Can not delete Scheduler";
+char *msg_sched_priv_exists = "Another Sched object also has same value for";
+char *msg_sched_logs_exists = "Another Sched object also has same value for";
```

Comment:
These values are same. But interesting thing is the actual attribute is appended to this message when this particular error is thrown. Also for code readability we used separate variable for the above messages even though they look similar.  Please look at the text marked bold in the following command output.
      Qmgr: c sched sc1 sched_priv=/tmp
      Qmgr: c sched sc2 sched_priv=/tmp
      qmgr obj=sc2 svr=default: Another Sched object also has same value for **sched_priv**
      qmgr: Error (15215) returned from server

We are in the process of adding these new messages/error codes etc to the EDD. We will make sure that we will add to it.